addons/mail/static/src/core/common/composer_model.js

```diff
@@ -24,14 +26,20 @@ export class Composer extends Record {
     mentionedChannels = fields.Many("Thread");
     cannedResponses = fields.Many("mail.canned.response");
     isDirty = false;
-    text = fields.Attr("", {
+    composerText = fields.Attr("", {
         compute() {
             if (this.syncTextWithMessage) {
                 return convertBrToLineBreak(this.message.body || "");
             }
-            return this.text;
+            return this.composerText;
+        },
+        onUpdate() {
+            prettifyMessageContent(this.composerText).then((content) => {
+                this.composerHtml = content;
+            });
```

Comment:
Is it possible to sync the editor value here with the new html when it gets ready?
How is the sync done after your last push?